fdm-app/app/routes/farm.$b_id_farm.settings.tsx

```
@@ -1,36 +1,44 @@
     const b_id_farm = params.b_id_farm
     if (!b_id_farm) {
-        throw data("Farm ID is required", { status: 400, statusText: "Farm ID is required" });
+        throw data("Farm ID is required", {
+            status: 400,
+            statusText: "Farm ID is required",
+        })
     }
 
     // Get details of farm
-    let farm;
+    let farm
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)